tools/docker/builder/ubuntu/bionic/Dockerfile

```diff
@@ -26,5 +26,34 @@ RUN apt-get update && apt-get install -y \
     python3-yaml \
     uuid-runtime \
     valgrind \
-    vim \
-    && rm -rf /var/lib/apt/lists/*
+    vim
+# AMBRIORIX dependencies
+RUN \
+    apt-get install -y \
+    bison \
+    flex \
+    libevent-dev \
+    libyajl-dev \
+    repo && \
+    echo "deb http://cz.archive.ubuntu.com/ubuntu eoan main universe" | tee -a  /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get install -y \
+    liburiparser-dev && \
+    rm -rf /var/lib/apt/lists/*
+WORKDIR ambiorix
+# Fetch and intall Bus Agnostic API libs, applications
+# "sed" is required to change from ssh to https protocols
+# https allows anonymous download, when ssh doesn't
+RUN \
+    repo init -u https://gitlab.com/soft.at.home/ambiorix/ambiorix.git && \
+    sed -i "s|ssh://git@gitlab.com/soft.at.home/ambiorix/|https://gitlab.com/soft.at.home/ambiorix/|g" .repo/manifests/default.xml && \
+    repo sync && \
+    make install -C libraries/libamxc && \
```

Comment:
Maybe add a comment that there are dependencies between the libs and that they need to be built/installed in this specific order.

Reply:
There are lots of `&&` in the script, eventually one may fail.
What do we do in case of an error?

Reply:
Docker image build will be aborted.

Reply:
I have added commentary about specific order of `build` and `install`. Resolving.